setup.php

```diff
@@ -143,13 +143,24 @@ function cloneClicked() {
 <span class="text">
 
 <?php
+// Checks if the server's PHP version and the client's browser are compatible with OpenEMR:
+require_once(dirname(__FILE__) . "/common/compatibility/checker.php");
+
+$compatCheck = new Checker();
+$potentialWarnings = $compatCheck->getCompatibility();
+if (!empty($potentialWarnings)) {
+  echo "<script>alert('" . $potentialWarnings . "');</script>";
```

Comment:
This messes up the \\n\\n line break. Is this really needed since we control the string that will be shown in the alert? In other words, it wouldn't be likely that naughty characters would get into this alert.